specification/loadtestservice/resource-manager/Microsoft.LoadTestService.Draft/preview/2023-12-01-preview/loadtestservice.json

```diff
@@ -0,0 +1,1021 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LoadTest Client",
+    "version": "2023-12-01-preview",
+    "description": "LoadTest client provides access to LoadTest Resource and it's status operations.",
+    "x-typespec-generated": [
+      {
+        "emitter": "@azure-tools/typespec-autorest"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "Operations"
+    },
+    {
+      "name": "LoadTests"
+    },
+    {
+      "name": "Quotas"
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.LoadTestService.Draft/operations": {
```

Comment:
why is the path showing .Draft?

Reply:
This I added to distinguish between already published specification, so that it is easy to compare the two json. Will remove it in PR to Azure repo.